chrome/content/zotero/xpcom/quickCopy.js

```diff
@@ -91,7 +100,7 @@ Zotero.QuickCopy = new function() {
 			var urlPath = nsIURI.path;
 		}
 		catch (e) {
-			return Zotero.Prefs.get("export.quickCopy.setting");
+			return quickCopyPref;
 		}
```

Comment:
I know it's not your code, but since you're touching this, I don't think that could ever throw (even for protocols that these attributes wouldn't make sense for). @dstillman?